### ISSUE TITLE
Cloning submodules fails

### ISSUE THREAD

**Issue by idevlp** [ActivityWatch/aw-watcher-web#73]
```
git clone --recursive https://github.com/ActivityWatch/aw-watcher-web.git
Cloning into 'aw-watcher-web'...
remote: Enumerating objects: 433, done.
remote: Counting objects: 100% (97/97), done.
remote: Compressing objects: 100% (65/65), done.
Rremote: Total 433 (delta 49), reused 69 (delta 30), pack-reused 336
Receiving objects:  91% (395/433), 140.01 KiB | 80.00 KiB/s
Receiving objects: 100% (433/433), 161.87 KiB | 91.00 KiB/s, done.
Resolving deltas: 100% (201/201), done.
Submodule 'aw-client-js' (git@github.com:ActivityWatch/aw-client-js.git) registered for path 'aw-client-js'
Submodule 'media' (https://github.com/ActivityWatch/media.git) registered for path 'media'
Cloning into 'C:/Users/<user>/projects/nodejs/aw-watcher-web/aw-client-js'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:ActivityWatch/aw-client-js.git' into submodule path 'C:/Users/<user>/projects/nodejs/aw-watcher-web/aw-client-js' failed
Failed to clone 'aw-client-js'. Retry scheduled
```

**Comment by johan-bjareholt** [comment]
Something is wrong with your setup, something is probably wrong with your SSH hosts since it says "Host key verification failed."

Google that line and you'll probably find some solution.

**Comment by idevlp** [comment]
> Something is wrong with your setup, something is probably wrong with your SSH hosts since it says "Host key verification failed."
> 
> Google that line and you'll probably find some solution.

I don't have ssh setup and you don't need ssh to clone public repos with public submodules.

I have googled it and the solution is to switch submodules over https instead of ssh.

**Comment by ErikBjare** [comment]
>I don't have ssh setup and we don't need ssh to clone public repos with public submodules.
>
>I have googled it and the solution is to switch submodules over https instead of ssh.

You're correct! This is not ideal.

I merged your PR (#74) manually in https://github.com/ActivityWatch/aw-watcher-web/commit/35e15e34634c9cd122b201d910cce15dc02c2bfd
